app/controllers/users_controller.rb

```diff
@@ -0,0 +1,36 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def update
+    @user = User.find(params[:id])
+    @user.update(user_params)
+
+    redirect_to edit_user_path(@user)
+  end
```

Comment:
- [ ] In this method, you need conditionals, just like you did in the `create` action, if it is successful it saves, if not it renders `edit` again.